beerus_cli/tests/cli.rs

```diff
@@ -2126,6 +2129,97 @@ mod test {
         }
     }
 
+    /// Test the `get_state_update` CLI command.
+    /// Given starknet lightclient returns an error, when query get_state_update, then the error is propagated.
+    /// Error case.
```

Comment:
this is not an error case, please update comment

Reply:
[Fixed](https://github.com/keep-starknet-strange/beerus/pull/176/commits/5e3ca16cc560782e3b62dbfecfddf93ce6ee3848)

Reply:
My bad, [this is the commit](https://github.com/keep-starknet-strange/beerus/pull/176/commits/c58756a7d8da8af24e42aeae20da3f1ccb99a139#)